.eslintrc

```diff
@@ -2,13 +2,20 @@
     "env": {
         "node": true
     },
-    "extends": ["airbnb-base", "prettier"],
+    "extends": [
+        "airbnb-base",
+        "prettier"
+
+    ],
```

Comment:
```suggestion
    "extends": [
        "airbnb-typescript/base",
        "prettier"

    ],
```

Reply:
This will though require to add eslint-config-airbnb-typescript to our dev-dependencies